src/styles/main.css

```diff
@@ -87,10 +88,10 @@ body {
 
     #instructions {
         text-align: center;
-        font-size: 105%;
+        font-size: 150%;
```

Comment:
I just tested it and with this font doesn't fit to some mobile screen. Let's use 120%.